Assets/Scripts/Objects/GameResourcesStorage.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+public enum ResourcePack
+{
+    noResources,
+    firstBaseResources
```

Comment:
```suggestion
    NoResources,
    FirstBaseResources
```

Should be better to use Pascal Casing for names of enumeration types, as in [ .NET Design-Guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations)